##[pylyzer] failed /home/dmitry/progs/python/WISCourse/Python_assignments/day8/venv/lib/python3.10/site-packages/flask/globals.py 1736025949 1713
.___v_desugar_1 = pyimport "<failure>"
.<failure> = pyimport "<failure>"
.annotations: Never
.t = pyimport "typing"
.typing = pyimport "typing"

.___v_desugar_2 = pyimport "<failure>"

.ContextVar: Never
.___v_desugar_3 = pyimport "local"
.local = pyimport "local"
.LocalProxy: {werkzeug.local.LocalProxy}

._no_app_msg: {"Working outside of application context.\n\nThis typically means that you attempted to use functionality that needed\nthe current application. To solve this, set up an application context\nwith app.app_context(). See the documentation for more information."}
._cv_app: Never
ctx = pyimport "ctx"
.app_ctx: ctx.AppContext
app = pyimport "app"
.current_app: app.Flask
.g: Never
._no_req_msg: {"Working outside of request context.\n\nThis typically means that you attempted to use functionality that needed\nan active HTTP request. Consult the documentation on testing for\ninformation about how to avoid this problem."}
._cv_request: Never
.request_ctx: ctx.RequestContext
.request: Never
.session: Never
